end2end-tests/models-jackson/src/test/kotlin/com/cjbooms/fabrikt/models/jackson/DeepNestedPolymorphismTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.cjbooms.fabrikt.models.jackson
+
+import com.example.models.*
+import com.fasterxml.jackson.module.kotlin.jsonMapper
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class DeepNestedPolymorphismTest {
```

Comment:
This does not appear to test deep json trees containing nested polymorphic structures. It seems to test individual branches in isolation

Reply:
I think the deep nesting is on the isomorphism side not the object tree side.

Reply:
Gotcha, reading the spec I understand that now.

Reply:
Another plus for having actual json examples :)